### PR TITLE
Changed the energy output of a nuclear reactor to match sf guide value

### PR DESCRIPTION
**Pull request by lvoz2**
Changed the output value from 16 J/s to 500 J/s

### CODE REVIEW

**Review by Phoenix-Starlight — DISMISSED** [review]
Good catch!

File: pages/Reactors.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This provides an easy way to add Fuel/Coolant or take some of the byproduct.
 ## Reactor Types
 | Type                | Power  | Buffer | Fuel                                | Cooling                  | Side Effects                 |
 | :------------------ | :----: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
-| Nuclear Reactor     | 16 J/s |  1.0x  | Neptunium, Uranium, Boosted Uranium | Reactor Coolant Cells    | N/A                          |
+| Nuclear Reactor     | 500 J/s |  1.0x  | Neptunium, Uranium, Boosted Uranium | Reactor Coolant Cells    | N/A                          |
 | Nether Star Reactor | 40 J/s |  3.0x  | Nether Stars                        | Nether Ice Coolant Cells | Nearby Entities get Withered |
```
Comment (WalshyDev) [inline pages/Reactors.md:13]:
```suggestion
| Nuclear Reactor     | 500 J/s |  1.0x  | Neptunium, Uranium, Boosted Uranium | Reactor Coolant Cells    | N/A                          |
| Nether Star Reactor | 40 J/s  |  3.0x  | Nether Stars                        | Nether Ice Coolant Cells | Nearby Entities get Withered |
```

File: pages/Reactors.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This provides an easy way to add Fuel/Coolant or take some of the byproduct.
 ## Reactor Types
 | Type                | Power  | Buffer | Fuel                                | Cooling                  | Side Effects                 |
 | :------------------ | :----: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```
Comment (WalshyDev) [inline pages/Reactors.md:11]:
```suggestion
| Type                | Power   | Buffer | Fuel                                | Cooling                  | Side Effects                 |
| :------------------ | :----:  | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```

**Review by WalshyDev — DISMISSED** [review]
Table formatting

File: pages/Reactors.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ It has to be placed exactly 3 blocks above the reactor and it can be used to int
 This provides an easy way to add Fuel/Coolant or take some of the byproduct.
 
 ## Reactor Types
-| Type                | Power  | Buffer | Fuel                                | Cooling                  | Side Effects                 |
-| :------------------ | :----: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
-| Nuclear Reactor     | 16 J/s |  1.0x  | Neptunium, Uranium, Boosted Uranium | Reactor Coolant Cells    | N/A                          |
-| Nether Star Reactor | 40 J/s |  3.0x  | Nether Stars                        | Nether Ice Coolant Cells | Nearby Entities get Withered |
+| Type                | Power   | Buffer | Fuel                                | Cooling                  | Side Effects                 |
+| :------------------ | :----:  | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```
Comment (WalshyDev) [inline pages/Reactors.md:11]:
```suggestion
| :------------------ | :-----: | :----: | ----------------------------------- | ------------------------ | ---------------------------- |
```

**Review by WalshyDev — COMMENTED** [review]
I'm st00pid

**Comment by WalshyDev** [comment]
Sidenote: Should we be saying per second anything? Per tick would make sense and actually be true everywhere. This is true for _most_ servers but not all

**Comment by svr333** [comment]
That's a decent point, we should let this PR pass though and then maybe make an issue to address this?
That we change it everywhere we can find it

**Review by svr333 — APPROVED** [review]

**Pull request merged**